backlogger.py

```diff
@@ -236,10 +238,38 @@ def render_influxdb(data):
                 print("\n".join(line for line in render_influxdb(data)))
             else:
                 initialize_md(data)
+                all_good, rows, bad_queries = render_table(data)
                 with open("index.md", "a") as md:
-                    all_good, rows = render_table(data)
                     for row in rows:
                         md.write("|".join(row) + "\n")
+                if os.environ.get('STATE_FOLDER'):
+                    old_state_file = os.path.join(os.environ['STATE_FOLDER'], "state.json")
+                    if os.path.exists(old_state_file):
+                        # open state.json from last run, see if anything changed and send slack notification if needed
+                        with open(old_state_file, "r") as sj:
+                            state = json.load(sj)
+                        old_bad_queries = set(state["bad_queries"].keys())
+                        new_bad_queries = set(bad_queries.keys())
+                        fixed_queries = old_bad_queries - new_bad_queries
+                        broken_queries = new_bad_queries - old_bad_queries
```

Comment:
```suggestion
                        broken_queries = new_bad_queries - old_bad_queries > 0 ? True : False
```
What do you think about this approach. I think is a little more readable. Unless i missed it the values are not used in the code below

Reply:
So your suggestion is removing the line `fixed_queries = old_bad_queries - new_bad_queries` and changing the second line? In case you only suggested to change the second line then it's commonly considered bad style to use a ternary operator to give back explicit truthy values from a comparison when the comparison itself yields a truthy value

Reply:
I'd also keep it this way. Using the ternary operator is introducing an additional branch and avoiding that is more elegant in my opinion. One might want to change `if broken_queries:` to `if broken_queries > 0:` below to make the condition we want to test there more explicit.

Reply:
I'd keep it this way is I would consider it less readable. Also this is no valid python syntax.